pkg/scalers/github_runner_scaler.go

```diff
@@ -41,19 +39,19 @@ type githubRunnerScaler struct {
 }
 
 type githubRunnerMetadata struct {
-	githubAPIURL              string
-	owner                     string
-	runnerScope               string
-	personalAccessToken       *string
-	repos                     []string
-	labels                    []string
-	noDefaultLabels           bool
-	enableEtags               bool
-	targetWorkflowQueueLength int64
-	triggerIndex              int
-	applicationID             *int64
-	installationID            *int64
-	applicationKey            *string
+	GithubAPIURL              string   `keda:"name=githubApiURL, order=triggerMetadata;resolvedEnv, default=https://api.github.com"`
+	Owner                     string   `keda:"name=owner, order=triggerMetadata;resolvedEnv"`
+	RunnerScope               string   `keda:"name=runnerScope, order=triggerMetadata;resolvedEnv"`
+	PersonalAccessToken       string   `keda:"name=personalAccessToken, order=authParams, optional"`
+	Repos                     []string `keda:"name=repos, order=triggerMetadata;resolvedEnv, optional"`
+	Labels                    []string `keda:"name=labels, order=triggerMetadata;resolvedEnv, optional"`
+	NoDefaultLabels           bool     `keda:"name=noDefaultLabels, order=triggerMetadata;resolvedEnv, default=false"`
+	EnableEtags               bool     `keda:"name=enableEtags, order=triggerMetadata;resolvedEnv, default=false"`
+	TargetWorkflowQueueLength int64    `keda:"name=targetWorkflowQueueLength, order=triggerMetadata;resolvedEnv, optional, default=1"`
```

Comment:
Minor nit, optional is not needed with default values

```suggestion
	TargetWorkflowQueueLength int64    `keda:"name=targetWorkflowQueueLength, order=triggerMetadata;resolvedEnv, default=1"`
```

Reply:
Thanks, I've resolved it